src/main/java/sirius/db/mixing/FieldLookupCache.java

```diff
@@ -119,4 +128,13 @@ public <E extends BaseEntity<?>> Value lookup(BaseEntityRef<?, E> ref, String fi
     public <E extends BaseEntity<?>> Value lookup(BaseEntityRef<?, E> ref, Mapping field) {
         return lookup(ref.getType(), ref.getId(), field);
     }
+
+    /**
+     * Provides the value of cache.
+     *
+     * @return the value of cache
+     */
+    public Cache<String, Value> getCache(){
```

Comment:
**Optional:** Maybe use our ReflectionHelper to access the Cache directly? Most likely we dont want to make this accessible everywhere